src/core_ocean/shared/mpas_ocn_tracer_short_wave_absorption_jerlov.F

```diff
@@ -186,6 +190,23 @@ subroutine ocn_tracer_short_wave_absorption_jerlov_tend(meshPool, forcingPool, i
       !$omp end parallel
 #endif
 
+      if(config_enable_shortwave_energy_fixer) then
+#ifndef CPRPGI
+         !$omp parallel
+         !$omp do schedule(runtime) private(k, layerFraction)
+#endif
+         do iCell = 1, nCells
+            layerFraction = bottomlayerShortwaveTemperatureFlux(iCell) / (maxLevelCell(iCell) + 1.0E-15_RKIND)
+            do k=1,maxLevelCell(iCell)
+               tend(index_temperature, k, iCell) = tend(index_temperature, k, iCell) + layerFraction
+            end do
+         end do
+#ifndef CPRPGI
+         !$omp end do
+         !$omp end parallel
+#endif
+      end if
+
```

Comment:
@mattdturner could you look at my openmp pragmas here?  I'm not sure of them

Reply:
This looks fine.